test/test.js

```diff
@@ -3,7 +3,7 @@ import test from 'ava';
 import 'babel-register';
 import 'babel-polyfill';
 
-import AnkiExport, { SEPARATOR } from '../src/index';
+import ankiExport, { SEPARATOR } from '../src/index';
```

Comment:
I was talking about reverting `test/test.js` to the master (https://github.com/ewnd9/anki-apkg-export/blob/master/test/test.js), sorry for the confusion.